### PR TITLE
cdc: limit cdc event fetching speed to reduce RocksDB read load

### LINKED ISSUE #11390: CDC should limit network bandwidth consumption
While https://github.com/tikv/tikv/pull/11152 corrects cdc incremental scan speed limit, TiKV may consume more network bandwidth than before. It could affect online service.

We should add a configuration that limit cdc network bandwidth consumption.

| Before / Now |
| -- |
| image |
| image |

### PR DESCRIPTION
**Pull request by hicqu**
### What is changed and how it works?

Issue Number: Close #11390 

What's Changed:

```commit-message
None
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/cdc/src/endpoint.rs [hunk 1]
```diff
@@ -470,6 +476,7 @@ impl<T: 'static + CdcHandle<E>, E: KvEngine, S: StoreRegionMeta> Endpoint<T, E,
             tso_worker,
             timer: SteadyTimer::default(),
             scan_speed_limiter: speed_limiter,
```
Comment (zhangjinpeng1987) [inline components/cdc/src/endpoint.rs:478]: Please rename to scan_speed_limiter in L443 and make it more clearly.

**Review by zhangjinpeng1987 — APPROVED** [review]
rest LGTM

File: components/cdc/src/initializer.rs [hunk 2]
```diff
@@ -90,7 +90,9 @@ pub(crate) struct Initializer<E> {
     pub(crate) request_id: u64,
     pub(crate) checkpoint_ts: TimeStamp,
 
-    pub(crate) speed_limiter: Limiter,
+    pub(crate) scan_limiter: Limiter,
```
Comment (pingyu) [inline components/cdc/src/initializer.rs:93]: How about `scan_speed_limiter` & `scan_disk_speed_limiter` ? That looks more clear to me.

**Review by pingyu — COMMENTED** [review]
Rest LGTM~

File: components/cdc/src/initializer.rs (hunk 2 above)
Comment (hicqu) [reply components/cdc/src/initializer.rs]: I prefer `scan_speed_limiter` (for disk I/O) and `fetch_speed_limiter` (for total fetched bytes). The reason is we already have a configuration `incremental_scan_speed_limit`, which works based on I/O.

**Review by pingyu — APPROVED** [review]

**Pull request merged**